lib/tapioca/helpers/config_helper.rb

```diff
@@ -124,10 +122,9 @@ def validate_config_options(command_options, config_key, config_options)
           :object
         end
 
+        next build_error("invalid value for option `#{config_option_key}` for key `#{config_key}` - expected " \
+                         "`#{command_option.type.capitalize}` but found #{config_option_value_type.capitalize}") \
         unless config_option_value_type == command_option.type
```

Comment:
Maybe a nitpick but I think this needs a tidy. I understand after noticing the trailing `\` after `build_error(...)` that it works but it's a bit gnarly. Can we make it a bit clearer?
I can't remember a time I saw a statement modifier on a separate line like that and I don't really see how it's an improvement over the block form.

Reply:
I tried doing the block form, but `rubocop` was still flagging it, so I ended up going back to this. Will try updating and see if I can get it to pass `bin/style`.